assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss

```diff
@@ -58,13 +58,13 @@
 		display: flex;
 		flex-direction: column;
 		justify-content: space-between;
+		padding: 14px $grid-gap-desktop;
 
 		@media (min-width: $bp-desktop) {
 			align-items: center;
 			border-top: 1px solid $c-mercury;
 			flex-direction: row;
 			margin: 42px 0 0;
```

Comment:
Since the footer only contains one `Cancel` action, I don't think these flex attributes are needed. They may have been left over after copying the styles from the User Input V2 screen.

Following up on the last comment, removing the media query here, and applying only the `padding`, `border-top`, and `margin` styles to `.googlesitekit-ad-blocking-recovery__footer` seems to resolve the problem.

Reply:
That works great, thanks! 👍🏻